Makefile

```diff
@@ -0,0 +1,2 @@
+install:
+	pip install -e .
```

Comment:
It's a bit unconventional for the default target in a makefile to install the program. Since `setup.py` already provides the same functions that a Makefile would provide (see `./setup.py --help-commands`), what do you think about just ditching the Makefile?